src/main/java/org/apache/bcel/classfile/Code.java

```diff
@@ -259,6 +259,20 @@ public LocalVariableTable getLocalVariableTable() {
         return null;
     }
 
+    /**
+     * Gets the local variable type table attribute {@link LocalVariableTypeTable}.
+     * @return LocalVariableTypeTable of Code, if it has one, null otherwise.
+     * @since 6.7.1
```

Comment:
New Javadoc since tags should be for 6.8.0.

Reply:
done